### PR TITLE
Runetales

### PR DESCRIPTION
**Pull request by Unerwarteter**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [ ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ ] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by Unerwarteter** [comment]
add server

File: minecraft_servers/runetales/background.png
Comment (DoJapHD) [inline minecraft_servers/runetales/background.png:1]: Compress file

File: minecraft_servers/runetales/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/runetales/background@2x.png:1]: Compress file

File: minecraft_servers/runetales/manifest.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "direct_ip": "runetales.net",
+  "server_wildcards": [
+    "%.runetales.net"
+  ],
+  "supported_languages": [
+     "en",
+     "de"
+  ],
+  "social": {
+    "web": "https://runetales.net"
```
Comment (DoJapHD) [inline minecraft_servers/runetales/manifest.json:13]: Cannot reach website

File: minecraft_servers/runetales/manifest.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    },
+  "gamemodes": {
+    "WorldOfMinecraft": {
```
Comment (DoJapHD) [inline minecraft_servers/runetales/manifest.json:16]: JSON key has to be lowercase

File: minecraft_servers/runetales/manifest.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      "name": "WorldOfMinecraft", 
+      "color": "#008cff",
+      "versions": "1.20.1-1.20.4*"
+    },
+  "discord": {
+    "server_id": 974013720831361104,
+    "rename_to_minecraft_name": true
```
Comment (DoJapHD) [inline minecraft_servers/runetales/manifest.json:23]: Reserved for partners; change to false

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/runetales/background@2x.png
Comment (Unerwarteter) [reply minecraft_servers/runetales/background@2x.png:1]: done

File: minecraft_servers/runetales/background.png
Comment (Unerwarteter) [reply minecraft_servers/runetales/background.png:1]: done

File: minecraft_servers/runetales/manifest.json (hunk 1 above)
Comment (Unerwarteter) [reply minecraft_servers/runetales/manifest.json]: removed

File: minecraft_servers/runetales/manifest.json (hunk 3 above)
Comment (Unerwarteter) [reply minecraft_servers/runetales/manifest.json]: done

File: minecraft_servers/runetales/manifest.json (hunk 2 above)
Comment (Unerwarteter) [reply minecraft_servers/runetales/manifest.json]: done

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**